### PR TITLE
Tutorials updates

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

File: runtime/tutorials/set_up_environment.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Setting up your Environment"
+---
+
+Deno is a secure runtime for JavaScript and TypeScript that comes with a set of
+built-in tools and utilities. In this tutorial, we’ll walk you through setting
+up your development environment for Deno using Visual Studio Code (VSCode).
```
Comment (magurotuna) [inline runtime/tutorials/set_up_environment.md:7]:
```suggestion
up your development environment for Deno using Visual Studio Code (VS Code).
```

Very minor stuff... I've heard that VS Code is official abbreviation. I can't find clear source, but it looks like their official FAQ consistently uses VS Code https://code.visualstudio.com/docs/supporting/faq

Comment (thisisjofrank) [reply runtime/tutorials/set_up_environment.md]: Ooh, nice catch!

File: runtime/tutorials/init_project.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+expanding the functionality in main.ts or you could add new modules, create more
+complex functions, and integrate third-party libraries. As you go, you can
+enhance your project structure by organizing your code into directories like
+`src` for source files and `tests` for test files. Implement robust error
+handling and logging to make your application more resilient. Don’t forget to
+write comprehensive tests in `main_test.ts` to ensure your code is reliable. You
+can also explore Deno’s built-in tools for formatting, linting, and bundling
+your code.
+
+// links go in here ^
```
Comment (rbetts) [inline runtime/tutorials/init_project.md:120]: Todo

File: runtime/tutorials/hello_world.md [hunk 3]
```diff
@@ -4,47 +4,28 @@ oldUrl:
   - /runtime/manual/examples/hello_world/
 ---
 
-## Concepts
+In this tutorial we'll walk through how to create an run a simple hello world
```
Comment (thisisjofrank) [inline runtime/tutorials/hello_world.md:7]:
```suggestion
In this tutorial we'll walk through how to create and run a simple hello world
```

File: runtime/tutorials/importing_modules.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+5 + 3 = 8
+5 - 3 = 2
+```
+
+You should see the sum and difference of the numbers printed to the console.
+
+## Using Third-Party Modules
+
+In this tutorial we're going to use the
+[@luca/cases](https://jsr.io/@luca/cases) package convert text strings to
```
Comment (thisisjofrank) [inline runtime/tutorials/importing_modules.md:60]:
```suggestion
[@luca/cases](https://jsr.io/@luca/cases) package to convert text strings to
```

File: runtime/tutorials/importing_modules.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+import the `cases` package in your project.
+
+Update your `main.ts` file to import two of the methods from the `cases` package
+and use them to change the format of the string `hello world`:
+
+```typescript title="main.ts"
+import { camelCase, splitPieces } from "@luca/cases";
+
+console.log(camelCase("hello world")); // helloWorld
+console.log(splitPieces("hello world")); // helloWorld
```
Comment (thisisjofrank) [inline runtime/tutorials/importing_modules.md:77]:
```suggestion
console.log(splitPieces("hello world")); // ["hello", "World"]
```

File: runtime/tutorials/init_project.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+  return a + b;
+}
+
+// Learn more at https://deno.land/manual/examples/module_metadata#concepts
+if (import.meta.main) {
+  console.log("Add 2 + 3 =", add(2, 3));
+}
+```
+
+This files is a simple example of a module in Deno. You can learn more about
```
Comment (thisisjofrank) [inline runtime/tutorials/init_project.md:62]:
```suggestion
This file is a simple example of a module in Deno. You can learn more about
```

**Comment by thisisjofrank** [comment]
closing, updated in other PRs

**Pull request closed without merging**